docker/authzMock/index.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const http = require('node:http')
```

Comment:
we should fix these warnings (sorry for my terrible code). It might be best to just eslint ignore them if that's all that's needed as we don't want to add dependencies to this if it can be avoided